content/luau/concepts/tables/terms/isfrozen/isfrozen.md

```diff
@@ -0,0 +1,50 @@
+
+```shell
+---
+Title: '.isfrozen'
+Description: 'Returns a boolean based on whether the table is in read-only mode or not.'
+Subjects:
+  - 'luau .isfrozen function'
+Tags:
+  - 'Boolean'
+  - 'Table'
+CatalogContent:
+  - 'paths/computer-science'
+---
+```
+
+## Introduction
+
+The **isfrozen** function in the Luau programming language returms a boolean based on whether the table is in read-only mode or not. When a table is in read-only mode, it means that the contents can't be modified, as a measure of data protection.
+
+
+
+## Syntax
+
+```luau
+isfrozen(table);
+```
+
+The 'isfrozen()' function returns a boolean value, true if the table is in read-only mode and false if it's not.
+
+
+
+
+## Codebyte Example
```

Comment:
```suggestion
## Example
```
This isn't a codebyte. Codebytes are special code blocks that the reader can execute. It's only supported for some languages, and Luau isn't one of them.